src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala

```diff
@@ -38,6 +40,12 @@ class ModelSpec(
   def getOptions: java.util.Map[String, String] =
     mapAsJavaMap(options.getOrElse(Map.empty))
 
+  /** Provide access to pre-processor via py4j. It can return Null / None in python. */
```

Comment:
does Scala Option not map properly to python value?

Reply:
py4j passes a java class, which somehow make it not accessible directly as "ModelSpec.name" and etc.